centreon/packages/ui/src/ThemeProvider/index.tsx

```diff
@@ -43,23 +43,38 @@ const getInputBaseRootStyle = ({ size }: InputBaseProps): CSSInterpolation => {
       minHeight: '36px'
     };
   }
+  if (equals(size, 'medium')) {
+    return {
+      minHeight: '40px'
+    };
+  }
+  if (equals(size, 'large')) {
+    return {
+      minHeight: '48px'
+    };
+  }
```

Comment:
Can you add a story that implement a large TextField ? You'll probably change the type of this prop: https://github.com/centreon/centreon/blob/develop/centreon/packages/ui/src/InputField/Text/index.tsx#L60